.gitignore

```diff
@@ -8,3 +8,4 @@ node_modules
 /build
 /bin/local-env/data
 phpunit.xml
+yarn.lock
```

Comment:
Did you accidentally run `yarn install` and then decided to ignore just in case? 🙂 
Might be a bit odd to have that here.

Reply:
That's what happened, mostly because my other projects use `yarn` but I think we can remove if necessary